GG/GG/GUI.h

```diff
@@ -376,6 +376,7 @@ class GG_API GUI
     //@}
 
     static GUI*  GetGUI();                  ///< allows any GG code access to GUI framework by calling GUI::GetGUI()
+    static void  PreRenderWindow(Wnd* wnd); ///< PreRender \p wnd's children and the \p wnd.
```

Comment:
explain what PreRenderWindow does, not just that is does itself on a Wnd's children...